frontend/views/containers/settings/UserProfile.vue

```diff
@@ -129,19 +129,22 @@ import L from '@view-utils/translations.js'
 
 const url = helpers.regex('url', /^(https?|ftp):\/\/[^\s/$.?#].[^\s]*$/i)
 
+// TODO: rename this file to match the name below!
```

Comment:
You can rename now ;)

Reply:
Thanks! Made a [note to self](https://github.com/okTurtles/group-income-simple/pull/611#issuecomment-521395410) in the PR and will have a closer look at the difference between the two `UserProfile.vue` files before merging.